### ISSUE TITLE
performance against Stable-Diffusion-Android (ShiftHackZ)

### ISSUE THREAD

**Issue by Foul-Tarnished** [ZTMIDGO/Android-Stable-diffusion-ONNX#19]
Hi

How does this compare to :
https://github.com/ShiftHackZ/Stable-Diffusion-Android 
in terms of performance ?

At least you can choose another model here easily (except the convert part), no hardcoded link to get the default model in the app

**Comment by Foul-Tarnished** [comment]
Apparently he got NNAPI to work, so I guess his project is faster when NAPI is enabled in app settings ?

**Comment by ZTMIDGO** [comment]
Yes, it is possible to use nnapi, but it doesn't seem to have much effect
